bundles/org.openhab.binding.luftdateninfo/README.md

```diff
@@ -0,0 +1,113 @@
+# LuftdatenInfo Binding
+
+<img style="float: right;" src="doc/logo-rund.png">
+Binding for the Sensor<i>Community <a href=https://luftdaten.info/>luftdaten.info</a>. The community provides instructions to build sensors on your own and they can be integrated into the database.
+With this binding you can integrate your sensor, a sensor nearby or even any sensors you want into openhab.
+
+## Supported Things
+
+Three Things are supported
+
+* Particulate Sensor - measure particulate matter PM2.5 and PM10
+* Conditions Sensor - measures environment conditions like temperature, humidity and some also provides atmospheric pressure
+* Noise Sensor - measures noise exposures in the environment
+
+## Discovery
+
+There's no auto discovery. See Thing configuration how to setup a Sensor.
+
+## Thing Configuration
+
+| Parameter       | Description                                                          |
+|-----------------|----------------------------------------------------------------------|
+| sensorid        | Sensor ID obtained from https://deutschland.maps.sensor.community/   |
+
+Perform the following steps to get the appropriate Sensor ID
+
+* Go to to [luftdaten.info map](https://deutschland.maps.sensor.community/)
+* Choose your wanted value in bottom list - now only the Sensors are displayed which are supporting this
+* Click on your / any Sensor and the ID is displayed in the top right corner.Note: Sensor ID is just the number without beginning hash #
+* Enter this Sensor ID into the thing configuration
+
+![Luftdaten.info Logo](doc/LuftdatenInfo-Map.png)
+
+## Channels
+
+### Particulate Sensor 
+
+| Channel ID           | Item Type            | Description                              |
+|----------------------|----------------------|------------------------------------------|
+| pm25                 | Number               | [Ultrafine particulates](https://en.wikipedia.org/wiki/Particulates#Size,shapeandsolubilitymatter) microgram per cubic meter |
+| pm100                | Number               | [Coarse particulate matter](https://en.wikipedia.org/wiki/Particulates#Size,shapeandsolubilitymatter) microgram per cubic meter  |
+
+### Conditions Sensor 
+
+| Channel ID           | Item Type            | Description                              |
+|----------------------|----------------------|------------------------------------------|
+| temperature          | Number               | current temperature in degrees Celsius |
+| humidity             | Number               | current humidity percent  |
+| pressure             | Number               | Atmospheric Pressure in hpa (not supported by all sensors) |
+| pressure-sea         | Number               | Atmospheric Pressure on sea level in hpa (not supported by all sensors)  |
+
+
+### Noise Sensor 
+
+Currently in [beta phase of the community](https://luftdaten.info/einfuehrung-zum-laermsensor/)
```

Comment:
This statement could be removed, since it will be outdated when the beta phase ends.

Reply:
True - line removed